bgpd/bgp_vty.c

```diff
@@ -322,6 +322,44 @@ int bgp_vty_find_and_parse_afi_safi_bgp(struct vty *vty,
 	return *idx;
 }
 
+
+int bgp_vty_find_and_parse_bgp(struct vty *vty,
+			       struct cmd_token **argv, int argc,
+			       int *idx, struct bgp **bgp)
+{
+	char *vrf_name = NULL;
+
+	assert(bgp);
+
+	if (argv_find(argv, argc, "view", idx)
+	    || argv_find(argv, argc, "vrf", idx)) {
+		vrf_name = argv[*idx + 1]->arg;
```

Comment:
this is pointless, you can just do
`argv_find(argv, argc, "VIEWVRFNAME", idx)`

Reply:
ok